bundle/manifests/opentelemetry.io_opentelemetrycollectors.yaml

```diff
@@ -3,7 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     cert-manager.io/inject-ca-from: opentelemetry-operator-system/opentelemetry-operator-serving-cert
-    controller-gen.kubebuilder.io/version: v0.6.0-beta.0
+    controller-gen.kubebuilder.io/version: v0.4.6
```

Comment:
@ekarlso You need to update your local `kubebuilder` version you are changing the version from `0.6.0` to `0.4.6`I created a test PR on top of your commit you can use it as reference here is the [commit](https://github.com/open-telemetry/opentelemetry-operator/pull/489/commits/a6f64017450091cff982db6092786dfec956464a) and the CI is successful.